syntax = "proto3";

package  lgbsttracker.sensors;

import "google/protobuf/timestamp.proto";

message LightSensor {
    string name = 1;
    float value = 2;
    google.protobuf.Timestamp creation_time = 3;
    google.protobuf.Timestamp last_updated_time = 4;
}